checkpoint1/AppointmentTracker/AppointmentTracker/Views/ServiceProvider/Details.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@model AppointmentTracker.Models.ServiceProviderModel
+@using AppointmentTracker.Services;
+
+@{
+    ViewData["Title"] = "Details";
+}
+
+<h2>Provider Details</h2>
+
+<div>
+    <hr />
+    <dl class="dl-horizontal">
+        <dt>
+            @Html.DisplayNameFor(model => model.Id)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Id)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.Name)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Name)
+        </dd>
+    </dl>
+    <br />
+    <h4><strong>@Html.DisplayFor(model => model.Name)'s Appointments</strong></h4>
+    <table class="table">
+        <thead>
+            <tr>
+                <th>
+                    Date
+                </th>
+                <th>
+                    Client
+                </th>
+                <th>
+                    Service
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach (var item in ServiceProviderRepository.GetAppointmentsForProvider(Model.Id))
```

Comment:
This is an example of mixing view and modal related logic. I would have attached this info to the model for this view instead of using the repository directly in my view